packages/flame/lib/src/extensions/matrix4.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:vector_math/vector_math_64.dart';
+
+export 'package:vector_math/vector_math_64.dart' hide Colors;
+
+extension Matrix4Extension on Matrix4 {
+  /// A first row and first column value.
+  double get m11 => storage[0];
```

Comment:
Isn't it better to teach the users to use the `entry(int row, int column)` method?
I think that gives more clear code.

Reply:
This extension was based on a conversation me and @luanpotter had a few months ago about adding more unity/godot like getters to the Matrix4 class. Because whenever you google matrix4 based code you get examples that are based on `mYX` properties. So to make it easier for people to work with the Matrix4 I came with the idea to add them as getters.

Reply:
Okay, might make sense... I didn't get up much of that when I googled, seems to be different ways of getting entries in different engines and languages. Can you send some examples? 
I stumbled upon one, and that one started on `m00` at least. I think if later there are examples and tutorials specifically for Flutter and Dart it will be confusing since there it would use `entry` and our users would have a higher chance of getting up such docs, on the other hand that would work too. Also imho it feels a bit old school/C++ to have entries like this.

Reply:
http://rbwhitaker.wikidot.com/monogame-basic-matrices This weirdly enough came up as first when I googled for unity.. But it does showcase it at the `Custom Matrices` part.

Reply:
if storage is a 1d array I think an `entry(i, j)` method alongside the m-getters is also useful

Reply:
That method already exists.